app/gui2/src/components/widgets/FileBrowserWidget.vue

```diff
@@ -114,17 +107,17 @@ function chooseFile(file: FileAsset) {
 }
 
 const isBusy = computed(
-  () => isPending.value || (selectedFile.value && selectedFileDetails.isPending.value),
+  () => isPending.value || (selectedFile.value && currentUser.isPending.value),
 )
 
 const anyError = computed(() =>
   isError.value ? error
-  : selectedFileDetails.isError.value ? selectedFileDetails.error
+  : currentUser.isError.value ? currentUser.error
   : undefined,
 )
 
-watch(selectedFileDetails.data, (details) => {
-  if (details) emit('pathSelected', details.file.path)
+watch(selectedFile, (file) => {
+  if (file && currentPath.value) emit('pathSelected', `${currentPath.value}/${file.title}`)
```

Comment:
This will not work if a file is selected before `currentPath` becomes ready

Reply:
If currentPath is not ready, which is only case when currentUser is not ready, we should be in "busy" state.

But fixed this anyway, as it is now more error-proof.

Reply:
Pending currentUser only causes busy state once a selection has been made.

Reply:
Oh, you are right! The better I wrote the fix.